ios/sidedishApp/sidedishApp/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="17701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="YOV-Ag-xwc">
```

Comment:
안녕하세요, 이번엔 5조 리뷰어가 된 제이슨이라고 합니다. 반갑습니다 😀
본격적으로 리뷰를 드리기 전에 Storyboard Entry Point가 Detail View Controller 로 설정되어 있어 화면을 볼수가 없네요 ㅎㅎ
이것만 먼저 수정 부탁드립니다.

관련 warning 
> Main.storyboard: warning: Unsupported Configuration: “Sidedish View Controller“ is unreachable because it has no entry points, and no identifier for runtime access via -[UIStoryboard instantiateViewControllerWithIdentifier:].

Reply:
안녕하세요🙇‍♀️

Reply:
수정하였습니다~!

Reply:
넵, 감사합니다 🙇‍♂️
하나만 더 부탁드릴게요!  swift 용 .gitignore 파일 추가해주세요.
xcuserdata 관련 파일들이 untracked files로 보여서요.
xcuserdata는 소스를 주고 받는데 불필요한 디렉토리이기 때문에 .gitignore 파일 추가 제안드립니다.

<img width="800" alt="스크린샷 2021-04-22 오전 8 52 58" src="https://user-images.githubusercontent.com/38216027/115636191-57be6380-a348-11eb-9af3-a9b9e2611541.png">

이미 push 원격저장소에도 .gitignore 적용하는 방법은 [이 블로그글](https://cjh5414.github.io/gitignore-update/)을 참고하시면 될 것 같습니다.

Reply:
네 수정하였습니다◡̈

Reply:
제이슨 반갑습니다! 잘 부탁드립니다 ㅎㅎㅎ

Reply:
아 넵 만사님 저도 잘 부탁드립니다!
V님도 잘 부탁드립니다 💪